parquet/src/test/scala/magnolify/parquet/ParquetTypeSuite.scala

```diff
@@ -185,14 +185,9 @@ class ParquetTypeSuite extends MagnolifySuite {
 
 case class Unsafe(c: Char)
 case class ParquetTypes(b: Byte, s: Short, ba: Array[Byte])
-case class Decimal32(bd: BigDecimal)
-case class Decimal64(bd: BigDecimal)
-case class DecimalFixed(bd: BigDecimal)
-case class DecimalBinary(bd: BigDecimal)
+case class Decimal(bd: BigDecimal, bdo: Option[BigDecimal])
```

Comment:
@shnapz here, added the `Optional[BigDecimal]`
`List[BigDecimal]` would have failed too